package.json

```diff
@@ -36,8 +36,10 @@
     "babel-runtime": "^6.20.0",
     "earcut": "^2.1.1",
     "flatbush": "^1.3.0",
+    "http-server": "^0.11.1",
     "js-priority-queue": "^0.1.5",
     "jszip": "^3.1.3",
+    "puppeteer": "^1.3.0",
```

Comment:
these 2 dependencies should be dev dependencies.

Reply:
`http-server` isn't needed so I'll remove it.

And `puppeteer` won't be installed by default (even in dev dependencies) to avoid download failures.

Fixing that.